### ISSUE TITLE
Mangadex API v5 Support

### ISSUE THREAD

**Issue by 19wolf** [Glagan/MyMangaDex#74]

**Comment by Glagan** [comment]
I don't know if I will add support for the future version of MangaDex, and it's API.

If I can't easily make MyMangaDex work with the new MangaDex version I will just archive this repository and redirect to SyncDex.

If you don't know, a new "version" is already out: [SyncDex](https://github.com/Glagan/SyncDex), it has the same (and more) features, and I'm already trying to update it so I can make it work as fast as possible once MangaDex re-opens.  
When I made SyncDex I made sure that it had at least all of the features that MyMangaDex had before publishing it, to make the old version obsolete and "upgrade" easily to the new version.  
The only "problem" was the stability, altough both were (are, and will be) unstable, but it shouldn't be a problem now (well, we'll see once the new updates roll out).

I might still update MyMangaDex if somehow the old (MangaDex) front end is updated to use the new API and somebody, somehow, make it available, but I highly doubt it will happen.

**TL;DR**: I probably won't update MyMangaDex and invite you to use [SyncDex](https://github.com/Glagan/SyncDex) instead.
